### ISSUE TITLE
SUI Breadcrumbs should have better keyboard navigation

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/terminal#12325]
### Windows Terminal version

1.13.10321.0

### Windows build number

22548.1002

### Other Software

None

### Steps to reproduce

(Part 1)
1. Open Settings UI
2. Go to a profile page
3. Press <kbd>Tab</kbd> repeatedly

(Part 2)
4. From a profile page, now open "Advanced" or "Appearance"
5. Press <kbd>Tab</kbd> repeatedly

### Expected Behavior

The right-most breadcrumb should not be a tab stop or invokable (or if it is invoked, it shouldn't do anything)

### Actual Behavior

Invoking "Advanced" or "Appearance" takes you back to the main profile page. 

### Additional Information
There's a chance that this (or some of this) should be filed on XAML. The goal here is to match what the settings app does.
